docs/xbcloud-binary-fifo-datasink.md

```diff
@@ -0,0 +1,58 @@
+# FIFO datasink
+
+[Percona XtraBackup 8.0.33-28](release-notes/8.0/8.0.33-28.0.md) implements a new datasink that uses `FIFO` (named pipes). With FIFO datasink feature, you can benefit from faster backups by streaming data in parallel from XtraBackup to `xbcloud`.
+
+## Stream to object storage
+
+When taking a backup, you can save the files locally or stream the files to either different servers or object storage. 
+
+Streaming backups to object storage using [`xbcloud`](xbcloud-binary-overview.md) works as follows:
```

Comment:
The verbs in the list should be the same
this one should be stream and use

Reply:
Updated